kitsune/community/static/js/ContributorsList.jsx

```diff
@@ -0,0 +1,28 @@
+import {Paginator} from './contributors-common.jsx';
```

Comment:
More particularly, this is actually using (or rather, being inspired by) object destructing assignment. This is the intermediate, which is equivalent to both the code in the file, and the code in my last comment. This kind of destructing pulls certain names out of an object, and binds them to variables. It can also be used in a bunch of other places.

```js
import _contributors_common from './contributors-common.jsx';
const {Paginator} = _contributors_common;
```